script.js

```diff
@@ -0,0 +1,104 @@
+import {languageDetailsMap} from './details.js';
+import {isNullOrUndefinied} from './checker.js';
+
+document.addEventListener("DOMContentLoaded", function() {
+
+    const RGB_BLACK_COLOR = "rgb(0, 0, 0)";
+    const RGB_YELLOW_COLOR = "rgb(255, 255, 0)";
+
+    const WRAPPER_ELEMENTS = Array.from(document.getElementsByClassName("wrapper"));
+    const PHP_IMAGE = document.getElementById("php");
+
+    let redPill = document.querySelector(".red");
+    if(!isNullOrUndefinied(redPill)) {
+        document.querySelector(".red").addEventListener("click", () => {
+            setDefaultProperties();
+            changeProperties();
+        });
+    }
+
+    let bluePill = document.querySelector(".red");
+    if(!isNullOrUndefinied(bluePill)) {
+        document.querySelector(".blue").addEventListener("click", () => {
+            setDefaultProperties();
+            setBlur(PHP_IMAGE);
+            changeBackgroundColor(PHP_IMAGE, RGB_YELLOW_COLOR)
+            setLanguageDetailsToElementByKey(PHP_IMAGE.id);
+        });
+    }
+
+
+    async function changeProperties() {
+        for (let i = 0; i <= WRAPPER_ELEMENTS.length; i++) {
+            let index = getRandomIndexWithRange(0, WRAPPER_ELEMENTS.length);
+            let element = WRAPPER_ELEMENTS[index];
+
+            changeBackgroundColor(element, RGB_YELLOW_COLOR);
+            await sleepMiliseconds(50);
+            changeBackgroundColor(element, RGB_BLACK_COLOR);
+
+            if(isLastElement(i)) {
+                changePropertiesForLastElement(element);
+            } 
+        }
+    }
+
+    function changePropertiesForLastElement(element) {
```

Comment:
To nie jest dla ostatniego elementu, a dla wskazanego. Jak podasz jako argument np. pierwszy, to przecież tez będzie działać jak należy - zmienia coś dla podanego. Więc nazwa do zmiany.